meson.build

```diff
@@ -2069,6 +2069,7 @@ libsystemd = shared_library(
         version : libsystemd_version,
         include_directories : libsystemd_includes,
         link_args : ['-shared',
+                     '-z', 'nodelete',
```

Comment:
A comment here might be good:

```
# Make sure our library is never deleted from memory, so that our open logging fds don't leak on dlopen/dlclose cycles.